src/compiler/compile/nodes/Element.ts

```diff
@@ -555,6 +555,69 @@ export default class Element extends Node {
 				});
 			}
 		}
+
+		if (handlers_map.has('click')) {
+
+			const a11y_interactive = new Set([
```

Comment:
Could you move this out to the top level?

Reply:
Done.